### ISSUE TITLE
[Crash] Vayra's l'interstellaire incompatible with AOTD VOK 2.3.0

### ISSUE THREAD

**Issue by RkShaRkz** [RkShaRkz/Starsector-IntelliJ-Template#46]
### Mod Name:
VayraMerged

### Mod Version:
Every version has this problem, it remained unsolved forever

### What were you doing when it happened?
doesn't matter, it happens when l'interstellaire tries to upgrade itself after AotD deleted all basic upgrade paths

### Steps to Reproduce:

1. use Popular Front
2. wait until l'interstellaire shows up
3. when it tries upgrading, the issue will happen

### Screenshots/Logs:
```
10012385 [Thread-3] ERROR com.fs.starfarer.combat.CombatMain  - java.lang.NullPointerException
java.lang.NullPointerException
    at com.fs.starfarer.api.impl.campaign.econ.impl.BaseIndustry.startUpgrading(BaseIndustry.java:572)
    at data.scripts.campaign.colonies.VayraColonialManager.pickNextUpgrade(VayraColonialManager.java:591)
    at data.scripts.campaign.fleets.VayraPopularFrontManager.advance(VayraPopularFrontManager.java:106)
    at com.fs.starfarer.campaign.CampaignEngine.advance(Unknown Source)
    at com.fs.starfarer.campaign.CampaignState.advance(Unknown Source)
    at com.fs.starfarer.BaseGameState.traverse(Unknown Source)
    at com.fs.state.AppDriver.begin(Unknown Source)
    at com.fs.starfarer.combat.CombatMain.main(Unknown Source)
    at com.fs.starfarer.StarfarerLauncher.super(Unknown Source)
    at com.fs.starfarer.StarfarerLauncher$1.run(Unknown Source)
    at java.lang.Thread.run(Thread.java:750)
```

### Additional info:
Do a proper fix, even if you have to check whether the mod is installed and *prevent* l'interstellaire from upgrading if it is.

**Comment by RkShaRkz** [comment]
Here's the ID that needs to be safeguarded against
```
"id":"aotd_vok"
```

maybe just don't upgrade the interstellar station if AOTK VOD mod is enabled.

**Comment by RkShaRkz** [comment]
should be fixed by #49 that was already merged in. Closing until it resurfaces.
